src/components/slideshow/slideshow.js

```diff
@@ -307,8 +307,7 @@ define(['dialogHelper', 'inputManager', 'connectionManager', 'layoutManager', 'f
          */
         function getSwiperSlideHtmlFromItem(item) {
             return getSwiperSlideHtmlFromSlide({
-                imageUrl: getImgUrl(item),
-                originalImage: getImgUrl(item, true),
+                originalImage: getImgUrl(item),
```

Comment:
you sure?

Reply:
imageUrl isn't used in the file anymore since we don't use the resized image and only use the originalImage.